spaces/S000174/properties/P000163.md

```diff
@@ -0,0 +1,7 @@
+---
+space: S000174
+property: P000163
+value: true
+---
+
+$|X| = \omega_1^\omega \leq (2^\omega)^\omega = 2^\omega = \mathfrak{c}$.
```

Comment:
I'd prefer to see more here (e.g. T404's Math StackExchange reference). But I think it's best to get everything merged in; I'll propose my own PR for a tweak or two for the community to consider.

Reply:
For posterity - this was perfectly fine, since the underlying set is now $\omega_1^\omega$.